packages/botonic-plugin-flow-builder/src/api.ts

```diff
@@ -15,24 +15,26 @@ import {
 import { FlowBuilderApiOptions } from './types'
 
 export class FlowBuilderApi {
-  private readonly url: string
-  public flow: HtFlowBuilderData
+  url: string
+  flow: HtFlowBuilderData
 
-  constructor(options: FlowBuilderApiOptions) {
-    this.url = options.url
-    if (options.flow) {
-      this.flow = options.flow
-    }
+  static async create(options: FlowBuilderApiOptions): Promise<FlowBuilderApi> {
```

Comment:
Just to know, why this function should be static?

Reply:
Static functions can be used to create new instances of objects. 

In this case I thought that it made no sense to be able to create a `FlowBuilderApi` instance and be able to call its methods without having the flow fetched. 
That's why I'm using this static method to ensure that every new `FlowBuilderApi` instance will have always the flow already fetched to be used.

I've added the `constructor` as a `private` class, this way we ensure this class won't be never created with the new keyword (meaning with an empty flow).

Reply:
For the reasons explained above.

If this method was not static, in order to call it, a `FlowBuilderApi` instance would be needed and that's what I'm trying to prevent by setting this method as static because otherwise you could be able to execute a `getNodeById` call after creating a new `FlowBuilderApi` instance (with the `new` method) without having the flow fetched.

Having this method as static I'm assuring that every `FlowBuilderApi` instance created (using the `create` method) will have already the flow fetched.

Reply:
Okay, I see thx!!